docs/home.html

```diff
@@ -22,31 +22,31 @@ <h4 class="latest-release subtle" ng-if="home.latestRelease">
   <div class="container">
     <ul class="featuring-links">
       <li>
-        <a href="#/docs/{{home.latestRelease.name}}/servicebuilder" title="gcloud-php docs" class="btn btn-docs">
+        <a href="#/docs/{{home.latestRelease.name}}/servicebuilder" title="google-cloud-php docs" class="btn btn-docs">
           <img src="src/images/icon-lang-php-gray.svg" alt="PHP icon" />
           Read the Docs
         </a>
       </li>
       <li>
-        <a href="https://github.com/GoogleCloudPlatform/gcloud-php" title="gcloud-php on GitHub" class="ext-link">
+        <a href="https://github.com/GoogleCloudPlatform/google-cloud-php" title="google-cloud-php on GitHub" class="ext-link">
           <img src="src/images/icon-link-github.svg" alt="GitHub icon" />
           GitHub
         </a>
       </li>
       <li>
-        <a href="https://github.com/GoogleCloudPlatform/gcloud-php/issues" title="gcloud-php issues on Github" class="ext-link">
+        <a href="https://github.com/GoogleCloudPlatform/google-cloud-php/issues" title="google-cloud-php issues on Github" class="ext-link">
           <img src="src/images/icon-link-github.svg" alt="GitHub icon" />
           Issues
         </a>
       </li>
       <li>
-        <a href="http://stackoverflow.com/questions/tagged/gcloud-php" title="gcloud-php on StackOverflow" class="ext-link">
+        <a href="http://stackoverflow.com/questions/tagged/google-cloud-php" title="google-cloud-php on StackOverflow" class="ext-link">
```

Comment:
This should be `http://stackoverflow.com/questions/tagged/google-cloud-platform+php`.